core/src/components/range/range.tsx

```diff
@@ -928,7 +946,7 @@ const renderKnob = (
     >
       {pin && (
         <div class="range-pin" role="presentation" part="pin">
-          {pinFormatter(value)}
+          {pinFormatter(value) + (theme === 'ionic' ? '%' : '')}
```

Comment:
What do you think about changing the `pinFormatter` prop to return `number | string`. That way, we don't have to add this theme specific logic, and the OS widget could provide its own formatter that would append the `%`

Reply:
Sounds great! I will do that